docs/src/components/app/style.scss

```diff
@@ -68,14 +68,46 @@
     padding: 2px;
     margin: 2px;
   }
+  .code-snippet {
+    font-variant: common-ligatures;
+    font-family: 'Fira Code', monospace;
+    background-color: #eee;
+    padding: 0.2rem;
+    margin: 0.1rem;
+    border-radius: 0.3rem;
+  }
+  code {
+    font-family: 'Fira Code', monospace;
+    padding: 0.2rem;
+    margin: 0.1rem;
+    border-radius: 0.3rem;
+  }
 }
 
 :global .mdc-theme--dark {
   .mdc-temporary-drawer li.mdc-list-item {
     color: #666;
   }
+  .code-snippet {
+    background-color: #696969;
+  }
 }
 
 :global .mdc-typography--body {
   line-height: 28px;
 }
+
+@font-face {
+  font-family: 'Fira Code';
+  src: url('https://cdn.rawgit.com/tonsky/FiraCode/1.205/distr/eot/FiraCode-Regular.eot');
```

Comment:
please ad a preload to these chrome's font in template.html

Reply:
As it's not necessary and work well without it I don't want it to be preloaded, but loaded after everything else, that's by design